lib/pdk/module/build.rb

```diff
@@ -83,17 +92,6 @@ def cleanup_build_dir
         FileUtils.rm_rf(build_dir, secure: true)
       end
 
-      # Clean up any files created during use of the PDK that shouldn't be part
-      # of the built module (e.g. test fixtures).
-      #
-      # @return nil
-      def cleanup_module
-        PDK::Util::Bundler.ensure_bundle!
-        PDK::Util::Bundler.ensure_binstubs!('rake')
```

Comment:
Does this need to re-appear somewhere?

Reply:
Nope, the purpose of this change is to remove the need for the in-module bundler environment when building release tarballs